inconspiquous/dialects/qu.py

```diff
@@ -25,6 +31,16 @@ class BitType(ParametrizedAttribute, TypeAttribute):
     name = "qu.bit"
 
 
+@irdl_attr_definition
+class QReg(ParametrizedAttribute, TypeAttribute):
```

Comment:
```suggestion
class RegisterType(ParametrizedAttribute, TypeAttribute):
```
We tend to end class names for types with `Type`.